packages/tensorboard/tensorboard.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tensorboard/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tensorboard"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tensorboard.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "--only-packages" "tensorboard" "--root" "." "-j" jobs "@install"]]
+depends: [
+  "base" {>= "0.11.0"}
+  "bigstringaf"
+  "stdio"
+  "ocaml" {>= "4.06"}
+  "ocamlfind" {build}
```

Comment:
What do you need ocamlfind for?

Reply:
I can't find any mention of ocamlfind in your source code

Reply:
Good catch, I just removed this thanks.